R/make_linter_from_regex.R

```diff
@@ -46,6 +50,8 @@ make_linter_from_regex <- function(regex,
         all_matches,
         line_numbers
       )
+
+      lapply(lints, function(x) x[vapply(x, length, integer(1L)) > 0L])
```

Comment:
Isn't `lengths()` available for a while now?

Reply:
thanks for making me check -- I know it's not in 3.1.

It's 3.2, which we use:

https://github.com/r-lib/lintr/blob/77558530416c4fb9017313abf14aae4c3b23d92a/R/backport_linter.R#L111-L115